data/language/english_uk.txt

```diff
@@ -4112,6 +4112,52 @@ STR_5803    :{SMALLFONT}{BLACK}Rotate selected map element
+STR_5810    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Sets UI scaling type
+STR_5811    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Use nearest neighbour scaling{NEWLINE}when window scaling factor set{NEWLINE}to integer values (1, 2, 3, etc)
+STR_5812    :{SMALLFONT}{BLACK}[Requires hardware display]{NEWLINE}Pause the game if Steam{NEWLINE}in-game overlay is opened
+STR_5813    :{SMALLFONT}{BLACK}Minimise the game if focus is{NEWLINE}lost while in fullscreen mode
+STR_5814    :{SMALLFONT}{BLACK}Changes the colour of the construction marker when building rides,{NEWLINE}paths, shops, scenery, etc.
+STR_5815    :{SMALLFONT}{BLACK}Cycle between day and night.{NEWLINE}Full cycle takes one in-game month
+STR_5816    :{SMALLFONT}{BLACK}Display banners in uppercase (RCT1 behaviour)
+STR_5817    :{SMALLFONT}{BLACK}Disables lightning effect{NEWLINE}during a thunderstorm
+STR_5818    :{SMALLFONT}{BLACK}Keep the mouse cursor in the window
+STR_5819    :{SMALLFONT}{BLACK}Invert right mouse dragging of the viewport
+STR_5820    :{SMALLFONT}{BLACK}Sets the color scheme used for the GUI
+STR_5821    :{SMALLFONT}{BLACK}Changes what measurement format is used for distances, speed, etc.
+STR_5822    :{SMALLFONT}{BLACK}Changes what currency format is used. Purely visual, there is no exchange rate implementation
+STR_5823    :{SMALLFONT}{BLACK}Changes what language is used
+STR_5824    :{SMALLFONT}{BLACK}Changes what format the{NEWLINE}temperature is displayed in
+STR_5825    :{SMALLFONT}{BLACK}Show height as generic units instead of measurement format set under "Distance and Speed"
+STR_5826    :{SMALLFONT}{BLACK}Changes what date format is used
+STR_5827    :{SMALLFONT}{BLACK}Select which audio device OpenRCT2 will use
+STR_5828    :{SMALLFONT}{BLACK}Mute the game if the window loses focus
+STR_5829    :{SMALLFONT}{BLACK}Select music to use on the main menu.{NEWLINE}Selecting RCT1 theme requires that you copy 'data/css17.dat' from your RCT1 game folder to 'data/css50.dat' in your RCT2 folder.
+STR_5830    :{SMALLFONT}{BLACK}Create and manage custom UI themes
+STR_5831    :{SMALLFONT}{BLACK}Show a separate button for the finance window in the toolbar
+STR_5832    :{SMALLFONT}{BLACK}Show a separate button for the research and development window in the toolbar
+STR_5833    :{SMALLFONT}{BLACK}Show a separate button for the cheats window in the toolbar
+STR_5834    :{SMALLFONT}{BLACK}Show a separate button for the recent news window in the toolbar
+STR_5835    :{SMALLFONT}{BLACK}Sort scenarios into tabs by their difficulty (RCT2 behaviour) or their source game (RCT1 behaviour)
+STR_5836    :{SMALLFONT}{BLACK}Enable scenario unlocking (RCT1 behaviour)
+STR_5837    :{SMALLFONT}{BLACK}Stay connected to a multiplayer server{NEWLINE}even if a desync or error occurs
+STR_5838    :{SMALLFONT}{BLACK}Adds a button for{NEWLINE}debugging tools to the toolbar.{NEWLINE}Access developer console with *KEY*
```

Comment:
The default key for displaying the developer console ```~``` is unfortunately used as an escape character in RCT2.
![screenshot 2016-04-25 00 46 33](https://cloud.githubusercontent.com/assets/1330770/14774446/33523574-0a84-11e6-8db0-dbd4e6c4a8b9.png)

I'm a bit unsure how to handle this.